codex-rs/apply-patch/src/lib.rs

```diff
@@ -40,6 +40,9 @@ pub enum ApplyPatchError {
     /// Error that occurs while computing replacements when applying patch chunks
     #[error("{0}")]
     ComputeReplacements(String),
+    /// A raw patch body was provided without an explicit `apply_patch` invocation.
+    #[error("patch detected without explicit call to apply_patch")]
```

Comment:
Should this include what the model should run instead to increase the chances it gets it right on the next turn?